packages/cli/scripts/pre-designed-nft.ts

```diff
@@ -170,7 +170,7 @@ function validateAttributesTypeIfExists(parent: string, attributes?: object) {
       }
 
       Object.keys(item).forEach((key) => {
-        if (!validNFTMetadataAttributeTypes.includes(typeof item[key])) {
+        if (!validNFTMetadataUriAttributeTypes.includes(typeof item[key])) {
```

Comment:
It appears that this variable does not exist, and after adding `scripts/**/*.ts` [here](https://github.com/alephium/alephium-web3/blob/a8803533be3f08945838679f1bc885139ba98784/packages/cli/tsconfig.json#L18), there are some other errors during the compilation of `pre-designed-nft.ts`.